src/getHangulAcronym.ts

```diff
@@ -0,0 +1,20 @@
+import { isHangul } from './_internal/hangul';
+
+/**
+ *
+ * @param getHangulAcronym
+ * @description
+ * 한글 문장을 입력받아서, 해당 한글 문장의 초성을을 리턴해줍니다.
+ * 한글 문장이 아닌, 문장은 취급하지않습니다. 추가로 한글 문장 + 영어 문장의 경우에도 취급하지않습니다.
+ */
+export function getHangulAcronym(text: string) {
+  if (!isHangul(text)) {
+    throw new Error('Invalid Hangul text, please input Hangul text only.');
+  }
+
+  const words = text.split(' ');
+
+  const firstHangulLetters = words.map(word => word.charAt(0));
+
+  return firstHangulLetters;
```

Comment:
한문장으로 작성해도 크게 가독성을 해치지 않을 것 같아요

```suggestion
return text.split(' ').map(word => word.charAt(0));
```